src/containers/ProfileDetails/components/AccessPoint/index.js

```diff
@@ -439,9 +441,13 @@ const AccessPointForm = ({
       <Card title="RF Enabled on This Profile">
         <Item name="rfProfileId">
           <Select
-            onPopupScroll={onFetchMoreRfProfiles}
+            onPopupScroll={e => onFetchMoreProfiles(e, 'rf')}
```

Comment:
add constants inside of `containers/ProfileDetails/constants/index.js` for the type strings